src/dbnode/storage/index/block.go

```diff
@@ -895,6 +897,11 @@ func (b *block) addAggregateResults(
 	// try to add the docs to the resource.
 	size, docsCount := results.AddFields(batch)
 
+	if err := b.aggDocsLimit.Inc(len(batch), source); err != nil {
+		// NB: this case should warn and continue.
+		b.logger.Warn("aggregate limit", zap.Error(err))
```

Comment:
Hm, this will get noisy. Can we just do a `_ = b.aggDocsLimit.Inc(...)` here?

Reply:
True, will do